Command/LoadDataFixturesDoctrineCommand.php

```diff
@@ -56,68 +57,58 @@ protected function configure()
 
   <info>php %command.full_name%</info>
 
-Fixtures are services that are tagged with doctrine.fixture.orm.
+Fixtures are services that are tagged with <comment>doctrine.fixture.orm</comment>.
 
-If you want to append the fixtures instead of flushing the database first you can use the <info>--append</info> option:
+If you want to append the fixtures instead of flushing the database first you can use the <comment>--append</comment> option:
 
-  <info>php %command.full_name% --append</info>
+  <info>php %command.full_name%</info> <comment>--append</comment>
 
-By default Doctrine Data Fixtures uses DELETE statements to drop the existing rows from
-the database. If you want to use a TRUNCATE statement instead you can use the <info>--purge-with-truncate</info> flag:
+By default Doctrine Data Fixtures uses DELETE statements to drop the existing rows from the database.
+If you want to use a TRUNCATE statement instead you can use the <comment>--purge-with-truncate</comment> flag:
+
+  <info>php %command.full_name%</info> <comment>--purge-with-truncate</comment>
 
-  <info>php %command.full_name% --purge-with-truncate</info>
 EOT
         );
     }
 
     protected function execute(InputInterface $input, OutputInterface $output)
     {
+        $ui = new SymfonyStyle($input, $output);
+
         /** @var $doctrine \Doctrine\Common\Persistence\ManagerRegistry */
         $doctrine = $this->getContainer()->get('doctrine');
         $em = $doctrine->getManager($input->getOption('em'));
 
-        if ($input->isInteractive() && !$input->getOption('append')) {
-            if (!$this->askConfirmation($input, $output, '<question>Careful, database will be purged. Do you want to continue y/N ?</question>', false)) {
-                return;
-            }
+        if (!$input->getOption('append')) {
+            $ui->ask('Careful, database will be purged. Do you want to continue y/N ?', false);
         }
 
         if ($input->getOption('shard')) {
             if (!$em->getConnection() instanceof PoolingShardConnection) {
-                throw new \LogicException(sprintf("Connection of EntityManager '%s' must implement shards configuration.", $input->getOption('em')));
+                $ui->error(sprintf(
+                    'Connection of EntityManager "%s" must implement shards configuration.',
+                    $input->getOption('em')
+                ));
+
+                return 1;
```

Comment:
I like the pretty errors that `$ui->error()` gives us, but I'd like to only use it for "normal errors" - error that happen as a result of somewhat "normal" use of the command. So, I'd prefer to keep the `LogicException` here (but I'm ok with the error below for "Could not find any fixture services to load")

Reply:
Ok 👍